colibre/auto_plotter/metallicity.yml

```
@@ -666,20 +665,21 @@ stellar_mass_star_fe_over_h_mol_hifloor_50:
   metadata:
     title: 'Stellar mass - [Fe/H]$_*$ relation (mean-of-log, 50 kpc aperture)'
     section: Stellar Metallicity
-    caption: 'Computed as the mass-weighted average of log [Fe/H]$_*$. The floor value is set to [Fe/H]=-3. All haloes are plotted, including subhaloes.'
+    caption: 'Computed as the mass-weighted average of log [Fe/H]$_*$. The floor value is set to [Fe/H]=-4. All haloes are plotted, including subhaloes.'
   observational_data:
     - filename: GalaxyStellarMassStellarMetallicity/Gallazzi2005_Data.hdf5
     - filename: GalaxyStellarMassStellarMetallicity/Kirby2013_Data.hdf5
-    - filename: GalaxyStellarMassStellarMetallicity/Fraser-McKelvie_2021.hdf5 
+    - filename: GalaxyStellarMassStellarMetallicity/Fraser-McKelvie_2021.hdf5
     - filename: GalaxyStellarMassStellarMetallicity/FIREbox.hdf5
     - filename: GalaxyStellarMassStellarMetallicity/Strom2022_FeH.hdf5
+    
 
-stellar_mass_star_fe_over_h_lom_50:
+stellar_mass_star_fe_over_h_mol_hifloor_50:
   type: "scatter"
   legend_loc: "upper left"
-  comment: "LoM"
+  comment: "MoL w. [F/H]=-4 floor"
```

You probably want here
```suggestion
  comment: "MoL w. [F/H]=-3 floor"
```